### PR TITLE
Fix the method error caused by null values ​​of prevState and data.lastUpdateStep

### PR DESCRIPTION
**Pull request by yangzehan**
10080806e7f768d58e770c7aa0bbdd7
e8dc94cce5ca74abeda56be9deb497b
image

### CODE REVIEW

**Comment by aiwenmo** [comment]
This branch has conflicts that must be resolved.

File: dinky-web/src/pages/DataStudio/BottomContainer/Console/ConsoleContent.tsx [hunk 1]
```diff
@@ -118,7 +129,7 @@ const ConsoleContent = (props: ConsoleProps) => {
         {node.status === JobStatus.FINISHED && (
           <CheckOutlined style={{ color: 'green', fontWeight: 'bold' }} />
         )}
-        {node.status === JobStatus.FAILED && (
+        {node.status == 'FAILED' && (
```
Comment (Zzm0809) [inline dinky-web/src/pages/DataStudio/BottomContainer/Console/ConsoleContent.tsx]: why change it??

**Review by aiwenmo — CHANGES_REQUESTED** [review]
Please rebase this branch onto dev branch.

File: dinky-web/src/pages/DataStudio/BottomContainer/Console/ConsoleContent.tsx [hunk 2]
```diff
@@ -85,16 +85,27 @@ const ConsoleContent = (props: ConsoleProps) => {
...
-        return data.lastUpdateStep;
-      } else if (!prevState || prevState.key === data.key) {
-        //未选择节点状态下选择根节点
-        return data;
+        // prevState和data.lastUpdateStep可能会有空值
+        if (prevState == undefined || data.lastUpdateStep == undefined) {
+          return data
+        }
+
+        if (prevState && prevState.key == data.lastUpdateStep.key) {
```
Comment (Zzm0809) [inline dinky-web/src/pages/DataStudio/BottomContainer/Console/ConsoleContent.tsx:93]: data.lastUpdateStep.key 可能为 undefined

**Pull request closed without merging**